File: Data Structures & Algorithms/two-integer-sum-ii/submission-0.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        
        int low = 0;
        int high = numbers.length -1;

        while(low < high)
        {
            int result = numbers[low] + numbers[high];

            if(result == target)
            {
                return new int[]{low+1,high+1};
            }
            else if(result > target )
            {
                high--;
            }
            else if(result < target)
            {
                low++;
            }
        }

        return new int[]{};
    }
}

File: Data Structures & Algorithms/two-integer-sum/submission-0.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:

        i = 0
        a = dict()
        for n in nums:
            if n in a :
                return [a[n], i]
            else:
                a[target-n] = i
            i = i + 1
            
        return list()
        